docs/openfaas-cloud/secrets.md

```diff
@@ -0,0 +1,45 @@
+OpenFaaS Cloud **secret** objects lets you store and manage sensitive information securely
+
+
+## Using secrets
+
+Secrets in OpenFaaS Cloud will be mounted as data volumes and made accessible to your function
+
+### Creating secrets
+
+To consume a **secret** in a function:
+
+1.\ Create the secret object `secrets.yml` using literals or from a file
+
+`faas-cli cloud seal --cert pub.cert --name {username}-database-creds --literal username=admin --literal password=1234`
+
+|                   |                         |
+|-------------------|-------------------------|
+| --name       | The name of the secret object prefixed with your GitHub username |
+| --cert       | The OpenFaaS cloud [public key](https://github.com/openfaas/cloud-functions/blob/master/pub-cert.pem). Please download to a local file |
+| --literal    | Secret key and value pair. You can specify this parameter more than once to add more secrets |
+| --from-file  | Read secret from file. **Note** secret key name will be the filename. At the moment we do not support specifying the key name when creating a secret from a file. |
+
+
+2.\ Update your function to include the secret in your `stack.yml`
+
+```yaml
+  my-function:   
+    secrets:
+      - database-creds
+```
+
+> **Note:** GitHub `{username}` prefix is removed in the `stack.yml`
+
+
+3.\ Update your function code to read the secret from `/var/openfaas/secrets/{key}`
+
+Where `{key}` is the name of the key pair. From our example in step 1 `username` or `password`:
+
+- `/var/openfaas/secrets/username`
+- `/var/openfaas/secrets/password` 
+
+If you are writing your code in Go, the OpenFaaS Cloud SDK provides a [ReadSecret](https://github.com/openfaas/openfaas-cloud/blob/master/sdk/secrets.go) method.
+
+4.\ Commit `secrets.yml` and other modified files. **Note** `secrets.yml` must be in the root of your GitHub repository
```

Comment:
What does the backslash do after 4.?

Reply:
The secrets.yml file should be placed at the / in the root of

Reply:
> The secrets.yml file should be placed at the / in the root of

Don't understand that re-wording?

Reply:
> What does the backslash do after 4.?

markdown escape to stop auto numbering of lists, it would reset them to 1. because of the code block between items